### DISCUSSION TITLE
【Q】皆さんのおすすめ書籍とその理由を

### DISCUSSION THREAD

**Discussion by Terralien-jp** [Hackathon5-TeamD/evolution-quest#36] [Q&A]
教えてください。

私はこれ

https://gihyo.jp/book/2021/978-4-297-12307-9

理由
授業で習ったことがナルホド！と理解できる丁度いい内容だったｗ

あとセキュリティつよつよな方が内容に間違いがないと言ってました（こっちがメイン）

**Comment by Rinasham** [comment]
https://www.amazon.co.jp/%E3%83%97%E3%83%AD%E3%82%B0%E3%83%A9%E3%83%A0%E3%81%AF%E3%81%AA%E3%81%9C%E5%8B%95%E3%81%8F%E3%81%AE%E3%81%8B-%E7%AC%AC%EF%BC%93%E7%89%88-%E7%9F%A5%E3%81%A3%E3%81%A6%E3%81%8A%E3%81%8D%E3%81%9F%E3%81%84%E3%83%97%E3%83%AD%E3%82%B0%E3%83%A9%E3%83%9F%E3%83%B3%E3%82%B0%E3%81%AE%E5%9F%BA%E7%A4%8E%E7%9F%A5%E8%AD%98-%E7%9F%A2%E6%B2%A2-%E4%B9%85%E9%9B%84-ebook/dp/B094J3CK1C/ref=nodl_

私これ🙋🏻‍♀️(リンク長...)

機械音痴な私がなんとなーくちょびっとだけパソコンの仕組みが分かったような気になれた本w
